README.md

```diff
@@ -4,10 +4,30 @@ Welcome to the Chengdu chapter of [ProtoSchool](https://proto.school)!
 
 We are a group of people passionate about the decentralized web and excited to learn, teach, and share ideas. Everyone is welcome!
 
+We focus on building IPFS ecosystem (Decentralized Storage Network) cooperate with [Protocol Labs](https://protocol.ai/).  
+
+Everyone is welcome to join us to learn, to preach, to research, to develop, to share ideas of the amazing tech together!   
+:raised_hands:
+## Preliminaries
+
+Everyone should read Code of Conduct below firstly ！
+
+Everyone should follow this [Chapter Chengdu Slides Template](https://docs.google.com/presentation/d/1itLIYBr_z9MgsueJFT2ckAJs56Xuq3T-LlG59kQljb4/edit)
+
+This repo support two language :[![](https://img.shields.io/badge/Lang-English-blue.svg)](README.md)  [![jaywcjlove/sb](https://jaywcjlove.github.io/sb/lang/chinese.svg)](README-zh.md)   
+
+## Meetup
+
+* [Chapter Chengdu Kickoff Meetup](https://docs.google.com/presentation/d/1itLIYBr_z9MgsueJFT2ckAJs56Xuq3T-LlG59kQljb4/edit)
+* [Chapter Chengdu Meetup - July 2019]()
```

Comment:
You're likely already aware of it, but this link is broken. I recommend using GitHub issues to announce new events and linking to them from your README.